### PR TITLE
chore(popover): adjust minWidth, add story

### PR DESCRIPTION
**Pull request by gloriliale**
Changed the maxWidth of popover to allow for more variety of content. Added a story to show this working.

### CODE REVIEW

File: packages/paste-core/components/popover/stories/index.stories.tsx [hunk 1]
```diff
@@ -30,6 +34,54 @@ export const Default: React.FC = () => {
...
+          </Stack>
+          <Separator orientation="horizontal" verticalSpacing="space40" />
+          <Stack orientation="horizontal" spacing="space40">
+            <Box width="10rem">
+              <Label htmlFor="date_picker_1">Start date</Label>
+              <DatePicker id="date_picker_1" />
+            </Box>
+            <Box width="8rem">
+              <Label htmlFor="time_picker_1">Start time</Label>
+              <TimePicker id="time_picker_1" />
```
Comment (TheSisb) [inline packages/paste-core/components/popover/stories/index.stories.tsx:76]: These ids are duped. Best to use useUID

**Review by TheSisb — APPROVED** [review]
Nice

**Review by zahnster — APPROVED** [review]
Looks good!

File: packages/paste-core/components/popover/stories/index.stories.tsx (hunk 1 above)
Comment (gloriliale) [reply packages/paste-core/components/popover/stories/index.stories.tsx]: this has been revamped to use useUID 👍

**Pull request merged**